settings/class-instant-articles-option-ads.php

```diff
@@ -185,19 +185,7 @@ public function sanitize_option_fields( $field_values ) {
 
 				case 'fan_placement_id':
 					if ( isset( $field_values['ad_source'] ) && 'fan' === $field_values['ad_source'] ) {
-						if (
-							! is_numeric( $field_values[ $field_id ] ) ||
-							strval( absint( $field_values[ $field_id ] ) ) !== $field_values[ $field_id ]
-						) {
-							add_settings_error(
-								$field_id,
-								'invalid_placement_id',
-								'Invalid Audience Network Placement ID provided'
-							);
-							$field_values[ $field_id ] = $field['default'];
-						} else {
-							$field_values[ $field_id ] = absint( $field_values[ $field_id ] );
-						}
+						$field_values[ $field_id ] = absint( $field_values[ $field_id ] );
```

Comment:
WordPress's [`absint()`](https://developer.wordpress.org/reference/functions/absint/) uses PHPs [`intval()`](http://php.net/manual/en/function.intval.php) which would truncate the value of the Placement ID at the underscore.

```php
intval('12345');  # 12345
intval('123_45'); # 123
```